File: ccc/2001/j1.java
//https://dmoj.ca/problem/ccc01j1
//CCC '01 J1 - Dressing Up

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int h = Integer.parseInt(s.next());
        for (int i = 0; i < Math.ceil(h/2)+1; i++) {
            System.out.println("*".repeat(2*(i+1)-1) + " ".repeat(2 * (h-(i*2)-1)) + "*" .repeat(2*(i+1)-1));
        }
        for (int i = (h/2)-1; i < Math.ceil(h/2)+1; i--) {
            if (i < 0) {
                break;
            }
            System.out.println("*".repeat(2*(i+1)-1) + " ".repeat(2 * (h-(i*2)-1)) + "*" .repeat(2*(i+1)-1));
        }
        s.close();
    }
}

File: ccc/2002/j1.py
#https://dmoj.ca/problem/ccc02j1
#CCC '02 J1 - 0123456789

n=int(input())

h=" * * *"
v0="*\n*\n*"
v2="      *\n      *\n      *"
v02="*     *\n*     *\n*     *"

if n==1:
    print()
    print(v2)
    print()
    print(v2)
    print()

elif n==2:
    print(h)
    print(v2)
    print(h)
    print(v0)
    print(h)

elif n==3:
    print(h)
    print(v2)
    print(h)
    print(v2)
    print(h)

elif n==4:
    print()
    print(v02)
    print(h)
    print(v2)
    print()

elif n==5:
    print(h)
    print(v0)
    print(h)
    print(v2)
    print(h)

elif n==6:
    print(h)
    print(v0)
    print(h)
    print(v02)
    print(h)

elif n==7:
    print(h)
    print(v2)
    print()
    print(v2)
    print()

elif n==8:
    print(h)
    print(v02)
    print(h)
    print(v02)
    print(h)

elif n==9:
    print(h)
    print(v02)
    print(h)
    print(v2)
    print(h)

elif n==0:
    print(h)
    print(v02)
    print()
    print(v02)
    print(h)
